src/indexing.jl

```diff
@@ -95,7 +95,7 @@ end
     end
 end
 
-# MAYBE: fixed-size indexing with bools? They would have to be Val's...
+# MAYBE: fixed-wx indexing with bools? They would have to be Val's...
```

Comment:
I hadn't committed the last changes. I've just pushed the version without `wx`. It is almost identical to the last version.